packages/task/src/browser/task-definition-registry.spec.ts

```diff
@@ -105,4 +152,54 @@ describe('TaskDefinitionRegistry', () => {
             expect(defs2!.taskType).to.be.eq(definitionContributionB.taskType);
         });
     });
+
+    describe('compareTasks function', () => {
+
+        beforeEach(() => registry.register(foobarTaskFixture.def));
+
+        it('should return false if given 2 task configurations with different type', () => {
+            const res = registry.compareTasks(
+                foobarTaskFixture.conf('id_1', 'type_1'),
+                foobarTaskFixture.conf('id_2', 'type_2'),
+            );
+            expect(res).to.be.not.ok;
+        });
+
+        it('should return true if given 2 same task configurations with empty arrays (different by reference) as custom property', () => {
+            const res = registry.compareTasks(
```

Comment:
I see the use of `res` a lot in the tests, what does it represnt? Perhaps a clearer name should be used?

Reply:
Nice suggestion. I agree and I meant "result" here. I'll change them.